### PR TITLE
Fix Insert With Keywords: add quotes around column names

### PR DESCRIPTION
**Pull request by remotelyliving**
Fixes https://github.com/behance/nbd.php-dbal/issues/25

@bryanlatten 
@ingluisjimenez 
@TJP85

### CODE REVIEW

File: src/AdapterAbstract.php [hunk 1]
```diff
@@ -532,6 +532,24 @@ protected function _quoteColumn( $column ) {
...
+   *
+   * @param array $columns
+   *
+   * @return array
+   */
+  protected function _quoteColumns( array $columns ) {
+
+    $quoted = [];
+
+    foreach ( $columns as $column ) {
```
Comment (ingluisjimenez) [inline src/AdapterAbstract.php]: maybe 

```
return array_map( function( $column ) {
  return $this->_quoteColumn( $column );
}, $colums );
```

**Comment by ingluisjimenez** [comment]
:+1:

File: src/AdapterAbstract.php (hunk 1 above)
Comment (remotelyliving) [inline src/AdapterAbstract.php]: if I want to run 90% slower ;-)

jk. Looks good, elegant even.

**Comment by bryanlatten** [comment]
Looking good

**Comment by bryanlatten** [comment]
:thumbsup

**Comment by bryanlatten** [comment]
:+1:

**Pull request merged**